packages/flame/lib/src/collisions/hitboxes/circle_hitbox.dart

```diff
@@ -28,8 +28,11 @@ class CircleHitbox extends CircleComponent with ShapeHitbox {
     required super.parentSize,
     super.angle,
     super.anchor,
+    bool isSolid = false,
```

Comment:
```suggestion
    super.isSolid,
```

Reply:
That's better. I copied how it has been done already in polygon_hitbox but I'll modify that too to use super instead.

Reply:
Actually dart doesn't let me use super for this. Perhaps because it's not a direct parameter of the parent?

Reply:
Ah, it's from the mixin. But you don't have to set it in the body anyways, you can set it after `:` and before super.

Reply:
I can't get that to work.

Reply:
I think it's because isSolid isn't a field in the class itself, even though it can be accessed in the body through this. Can we keep it as it is then.

Reply:
Alright, I thought it got the fields before evaluation those, let's keep it like it is then.